clueless/settings.py

```diff
@@ -118,3 +117,5 @@
 # https://docs.djangoproject.com/en/1.10/howto/static-files/
 
 STATIC_URL = '/static/'
+
+from .settings_secret import *
```

Comment:
That's so weird...I can pull the branch and test when I'm home to make sure it didn't break anything, but maybe it doesn't matter.